modules/core/src/lib/deck.js

```diff
@@ -536,6 +536,13 @@ export default class Deck {
     return views;
   }
 
+  _onContextLost() {
```

Comment:
Nice. Seems like we could also handle this inside AnimationLoop by checking if it has been destroyed.